404.html

```diff
@@ -0,0 +1,17 @@
+<html>
+<head>
```

Comment:
We should have a title. "The Prototype Kit training website has moved"?

Reply:
Good point. Any thoughts @NoraGDS?

Reply:
Yes, agree. thanks @domoscargin